docs/contributors/contributors-guide.md

```diff
@@ -0,0 +1,171 @@
+
+## Getting Started
+
+To get started, you will need to create an account on OpenSauced. You can do this by visiting the [app.opensauced.pizza](https://app.opensauced.pizza) and clicking the "Connect with GitHub" button. You will be prompted to sign in with your GitHub account. Once you have signed in, you will be redirected to the OpenSauced dashboard.
+
+## Highlights: Connecting and Recognizing Contributions
+
+Leveraging OpenSauced's [Highlights](../features/highlights.md) feature is a great way for you to showcase your valuable contributions to your favorite open source projects. You can curate and share your contribution achievements and milestones and bring visibility to your hard work.
+
+Highlights feature allows to:
+
+- list issues that you'd raised to spot bugs or request a feature,
+- share Dev.to blog posts you've written,
+- celebrate PRs for your project.
+
+### Creating a New Highlight
+
+There are two ways to create a new highlight:
+
+1. After clicking the [Highlights tab](https://app.opensauced.pizza/feed) in the top navigation, click the input that says "Post a highlight to show your work!"
+
+2. From your profile, click the input that says "Post a highlight to show your work!"
+
+After you've clicked the input, you'll see a form that allows you to create a new highlight.
+
+There are two ways to create a highlight:
+
+1. Paste the URL to your PR, Issue, or Dev.to blog post (shown in blue below).
+2. Choose from a list of suggestions (shown in orange below).
+
+![highlight suggestions](../../static/img/highlights.png)
+
+If you're adding a blog post, you'll need to associate it to a repository by clicking the "Add a repo" button.
+
+Finally, it's time to add the details of your highlight. You can add a description yourself or use the auto-summarize function to generate a description.
+
+![gif of creating a highlight](../../static/gif/highlight.gif)
+
+### Writing an Effective Highlight
+
+Writing an effective highlight is key to showcasing your contributions and skills, and attracting potential collaborators and recruiters. Here are some tips to help you write an effective highlight:
+
+#### Highlighting Issues
+
+1. **Summarize the problem**: Begin by briefly describing the problem the project is currently facing. This helps other contributors or recruiters gain a better understanding of the situation.
+2. **Describe the impact:** Briefly state how the problem negatively impacts the project. This would exemplify the issue's significance. 
+3. **Describe a possible solution**: Mention the solution that you are considering to use to solve the issue. This would help display your problem solving skills to recruiters.
```

Comment:
Hey @BekahHW! :) Here's what I came up with. I heavily based it on my experience with the product. What do you think?